src/GitVersionCore/Configuration/ConfigurationProvider.cs

```diff
@@ -73,10 +103,27 @@ static void MigrateBranches(Config config)
             // Map of current names and previous names
             var dict = new Dictionary<string, string[]>
             {
-                { "hotfix(es)?[/-]", new [] { "hotfix[/-]" }},
-                { "features?[/-]", new [] { "feature[/-]", "feature(s)?[/-]" }},
-                { "releases?[/-]", new [] { "release[/-]" }},
-                { "dev(elop)?(ment)?$", new [] { "develop" }}
+                {"hotfix(es)?[/-]", new[]
+                    {
+                        "hotfix[/-]"
+                    }
+                },
+                {"features?[/-]", new[]
+                    {
+                        "feature[/-]",
+                        "feature(s)?[/-]"
+                    }
+                },
+                {"releases?[/-]", new[]
+                    {
+                        "release[/-]"
+                    }
+                },
+                {"dev(elop)?(ment)?$", new[]
+                    {
+                        "develop"
+                    }
+                }
```

Comment:
Although this is formatted according to the ReSharper settings, I'd say the settings are wrong. :confused: Sorry, but can you please revert this back?